packages/headless/src/features/result-preview/result-preview-request-builder.ts

```diff
@@ -24,8 +24,8 @@ export function buildResultPreviewRequest(
     accessToken,
     organizationId,
     enableNavigation: false,
-    requestedOutputSize: 0,
     q,
     ...options,
+    requestedOutputSize: options.requestedOutputSize || 0,
```

Comment:
It would be good to include a unit test for this line, passing `undefined` and making sure that `0` is sent.

Reply:
I will add that test.